audio/BoCA/Portfile

```diff
@@ -37,6 +40,14 @@ depends_lib         port:expat \
 patchfiles          patch-set-install-path-runtime.diff \
                     patch-set-install-path-components.diff
 
+# Use ffmpeg7 if available
+set ffmpeg_v 7
+post-patch {
+    foreach f [glob ${worksrcpath}/scripts/boca_*coder_ffmpeg*.xml] {
+        reinplace "s|<command>ffmpeg</command>|<command>${prefix}/bin/ffmpeg${ffmpeg_v}</command>|g" $f
+    }
+}
+
```

Comment:
this doesn't make sense; you're changing it so that `ffmpeg7` is used but do not declare a dependency on it. That has to change.

Reply:
You can make matching variants for two ports. It is done with a few other ports, you may refer to some examples.

Reply:
Can you please elaborate? [MacPorts docs](https://trac.macports.org/wiki/PortfileRecipes#require_active_variants) say: _Depending upon a specific port variant is not possible in MacPorts_.

Reply:
You cannot depend on a _non-default_ variant by default in a sense that CI gonna fail (it is _still done_ when required, see `rtorrent` port). You _can_ require a non-default variant of a dependency for a non-default build.

Reply:
But this is exactly what `require_active_variants` does, no? Through this, in the `freac` portfile,  I can require `BoCA +ffmpeg7` when `freac +ffmpeg7` is being requested. But this means that someone that installs `freac +ffmpeg7` will face an error and then has to manually install `BoCA +ffmpeg7`.

There is no one - _ever_ - going to install `BoCA` without `freac`. So I think it is a pragmatic approach to leave the dependency configuration in `freac`.

My primary concern, however, is to get this PR through in reasonable time, so if my reviewers insist that `BoCA` needs a dependency declared on `ffmpeg` then I will do so.